format_specs/data_format.md

```diff
@@ -203,15 +95,27 @@ Column  Field       Description
 5       UUID        A 36-digit hexadecimal ID for the flow.
 ```
 
-### sectors.csv
+**sectors.csv**
+
 The file `sectors.csv` contains the metadata of the sectors in the model.
 It should have the following columns:
 
 ```
 Column  Field         Description
 0       Index         The zero-based matrix index of the sector.
-1       ID            The ID of the flow.
+1       ID            The ID of the sector.
 2       Name          The name of the sector, e.g., 'Wood products'.
 3       Code          The code of the sector in the Sector_Schema.
 4       Location      The location code.
 5       Description   A text description of the sector, optional.
+```
+
+**years.csv**
+
+The file `years.csv` contains the years for model data that is in model [sector-by-year](https://github.com/USEPA/useeior/blob/master/format_specs/model.md#sector-by-year) format. It should have the following columns:
```

Comment:
In the URL https://github.com/USEPA/useeior/blob/master/format_specs/Model.md#sector-by-year `model` should start with an upper-case letter; the format is index + year?

Reply:
Addressed in this commit. I agree linking to that data format was confusing.  I updated the metadata to describe that years is needed for the new Rho and Phi matrices which have columns as years.  That's why years.csv needed to be added to the metadata - so those matrix columns could be indexed.
https://github.com/USEPA/USEEIO_API/commit/0bd5d642fccded68316dfcd3d3b2dd8b2ce8ca28